package_additions.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# Create archive with docker additions for sprint (IDE and IRC).
+docker pull briangilbert/cloud9-alpine:20180318 linuxserver/thelounge:94
```

Comment:
It looks like on docker-17.12.0-ce docker pull only supports exactly 1 argument.

> "docker pull" requires exactly 1 argument.
> See 'docker pull --help'
>
> Usage: docker pull [OPTIONS] NAME[:TAG|@DIGEST] [flags]
> 
> Pull an image or a repository fro a registry

Reply:
have split into two commands now